README.md

```diff
@@ -61,23 +61,25 @@ ReactJS based Presentation Library
 
 ## Getting Started
 
-The new best way to get started is by running `create-react-app my-presentation --scripts-version spectacle-scripts`. This will use `create-react-app` to create almost everything you need. This however, doesn't include publish scripts, and ejecting is required for fancy stuff.
+There are three ways to get started on your presentation. The first way is to run the following commend in your terminal.
```

Comment:
`commend` => `command`

let's also maybe end this section with a `:` instead of a period if that sounds good!